src/containers/Tasks/ImportAttestation/ImportAttestation.tsx

```diff
@@ -1,78 +1,110 @@
 import * as sdk from '@kiltprotocol/sdk-js'
-import React from 'react'
-import { connect } from 'react-redux'
+import React, { useState, useEffect } from 'react'
+import { connect, MapStateToProps } from 'react-redux'
 import AttestedClaimsListView from '../../../components/AttestedClaimsListView/AttestedClaimsListView'
 import ClaimDetailView from '../../../components/ClaimDetailView/ClaimDetailView'
-
+import PersistentStore, {
+  State as ReduxState,
+} from '../../../state/PersistentStore'
 import { notifySuccess } from '../../../services/FeedbackService'
 import * as Claims from '../../../state/ducks/Claims'
 
+type StateProps = {
+  claims: Claims.Entry[]
+}
+
 type DispatchProps = {
   addAttestationToClaim: (attestation: sdk.IAttestedClaim) => void
 }
 
 type OwnProps = {
-  attestedClaim: sdk.IAttestedClaim
-
+  attestation: sdk.IAttestation
+  requestForAttestationRootHash: sdk.IRequestForAttestation['rootHash']
```

Comment:
Remove this and also remove it inside the `MessageDetailView.tsx` at `MessageBodyType.SUBMIT_ATTESTATION_FOR_CLAIM`